app/_redirects

```diff
@@ -886,30 +871,40 @@
+/gateway-oss/0.9.x/*                        https://legacy-gateway--kongdocs.netlify.app/gateway-oss/0.9.x/:splat
+/gateway-oss/0.8.x/*                        https://legacy-gateway--kongdocs.netlify.app/gateway-oss/0.8.x/:splat
+/gateway-oss/0.7.x/*                        https://legacy-gateway--kongdocs.netlify.app/gateway-oss/0.7.x/:splat
+/gateway-oss/0.6.x/*                        https://legacy-gateway--kongdocs.netlify.app/gateway-oss/0.6.x/:splat
+/gateway-oss/0.5.x/*                        https://legacy-gateway--kongdocs.netlify.app/gateway-oss/0.5.x/:splat
+/gateway-oss/0.4.x/*                        https://legacy-gateway--kongdocs.netlify.app/gateway-oss/0.4.x/:splat
+/gateway-oss/0.3.x/*                        https://legacy-gateway--kongdocs.netlify.app/gateway-oss/0.3.x/:splat
+/gateway-oss/0.2.x/*                        https://legacy-gateway--kongdocs.netlify.app/gateway-oss/0.2.x/:splat
+
+# ENTERPRISE DOC ARCHIVES
+
+/enterprise/0.31-x/*    https://legacy-gateway--kongdocs.netlify.app/enterprise/0.31-x/:splat
+/enterprise/0.32-x/*    https://legacy-gateway--kongdocs.netlify.app/enterprise/0.32-x/:splat
+/enterprise/0.33-x/*    https://legacy-gateway--kongdocs.netlify.app/enterprise/0.33-x/:splat
+/enterprise/0.34-x/*    https://legacy-gateway--kongdocs.netlify.app/enterprise/0.34-x/:splat
+/enterprise/0.35-x/*    https://legacy-gateway--kongdocs.netlify.app/enterprise/0.35-x/:splat
+/enterprise/0.36-x/*    https://legacy-gateway--kongdocs.netlify.app/enterprise/0.36-x/:splat
+/enterprise/1.3-x/*     https://legacy-gateway--kongdocs.netlify.app/enterprise/1.3-x/:splat
+/enterprise/1.5.x/*     https://legacy-gateway--kongdocs.netlify.app/enterprise/1.5.x/:splat
 
-# 3.0
-
-/gateway/2.8.x/install-and-run/                   /gateway/latest/support-policy/
-/gateway/2.8.x/install-and-run/centos/            /gateway/latest/support-policy/
-/gateway/latest/install/linux/os-support/         /gateway/latest/support-policy/
-/gateway/2.8.x/get-started/quickstart/            /gateway/latest/get-started/
-/gateway/2.8.x/get-started/quickstart/configuring-a-service/            /gateway/latest/get-started/services-and-routes/
-/gateway/2.8.x/get-started/quickstart/configuring-a-grpc-service/       /gateway/latest/get-started/ratelimiting/
-/gateway/2.8.x/get-started/quickstart/enabling-plugins/                /gateway/latest/key-concepts/plugins/
-/gateway/2.8.x/get-started/quickstart/adding-consumers/               /gateway/latest/get-started/key-authentication
-/gateway/2.8.x/admin-api/rbac/examples/                              /gateway/latest/admin-api/rbac/reference/
-
-# 3.1
-/gateway/latest/production/logging/     /gateway/latest/production/logging/log-reference/
-
-
-#3.2
+# COMMUNITY DOC REBRANDING
+/community   https://konghq.com/community-resources/
+/:version/*  https://legacy-gateway--kongdocs.netlify.app/gateway-oss/:version/:splat
```

Comment:
Unfortunately, `/:version/` can't replace [this block](https://github.com/Kong/docs.konghq.com/pull/5702/files#diff-a19a552735ad7e7a99a125b16220722ae6a771a68b5b06332243b6709a357b4bL321-L358)

`/:version/` matches any `/segment/` which is causing things like:
`/gateway/` to be redirected to `/gateway-oss/gateway/`

Reply:
yeah I learned this the hard way with this PR 😢